docs/modules/ROOT/pages/integrating-with-existing-deployments.adoc

```diff
@@ -30,23 +25,27 @@ Once you have completed all of the below steps you may need to restart Prometheu
 
 You will need to modify your existing Prometheus configuration file - this is typically named `prometheus.yml`.
 
-See our http://localhost:4040/cmos/0.1/tutorial-standalone.html#_install_prometheus[Install Prometheus] documentation for an example of what this file should look like with an appropriate Couchbase `scrape_config`.
+See our xref:tutorial-standalone.html#_install_prometheus[Install Prometheus] documentation for an example of what this file should look like with an appropriate Couchbase Server `scrape_config`.
```

Comment:
May be mistaken, but I thought xrefs needed to reference the `.adoc` file, not the resulting `.html`.

Reply:
Oops yeah this will be from a copy and paste from a preview, fixed